src/Hero.js

```diff
@@ -25,8 +25,12 @@ export default function Hero() {
             Resources, tooling, and design guidelines for building interfaces with GitHub’s design system
           </Text>
           <Text fontFamily="mono" as="p" color="blue.3" mt={5}>
-            <LinkLight fontSize={[0, 1, 2]} href="https://styleguide.github.com/primer/">
-              Style guide
+            <LinkLight fontSize={[0, 1, 2]} href="https://primer.style/css/">
+              Documentation
```

Comment:
Alternatively, this could be renamed to  _CSS_ to match the naming in _Primer CSS_ and _Primer Components_.

Reply:
```suggestion
              Team
```

Reply:
Hey @gtsiolis we decided on putting the team link here instead!

Reply:
@emplums sounds good! This can always be improved later. 🏀